src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -570,7 +570,7 @@ def get_versioned_sdk_path(api_profile, resource_type, operation_group=None):
 
 
 def get_versioned_sdk(api_profile, resource_type, *attr_args, **kwargs):
-    checked = kwargs.get('checked', True)
+    checked = kwargs.get('checked', None)
```

Comment:
This still overwrites the default `checked=False` of `_get_attr`. It is coincident that `False` and `None` have the same effect.

If the default is `checked=True`, it will be overwritten as `None`.